charts/newrelic-logging/Chart.yaml

```diff
@@ -16,3 +16,7 @@ maintainers:
 keywords:
   - logging
   - newrelic
+dependencies:
+  - name: common-library
+    version: 0.14.3
+    repository: file://../../library/common-library
```

Comment:
Missing newline at EOF

Reply:
@TheQueenIsDead, could you also update to the latest version of the common library, please?

Reply:
Merged the latest master and bumped the dependency version in 6e00fbc660652762fe41b5fb25e7628e7c7fdde7